### DISCUSSION TITLE
What's the correct way to edit a control template in WinUI3

### DISCUSSION THREAD

**Discussion by ReneeGA2020** [microsoft/microsoft-ui-xaml#8858] [Q&A] [answered]
I have a Combox which I wanted to change its control template. How do I generate the default template?

**Comment by DHancock** [comment, marked as answer]
The default templates can be found in a `generic.xaml` file contained in the WinAppSdk nuget package. If you've using 1.4.0, one can be found at:
````
C:\Users\<your user name>\.nuget\packages\microsoft.windowsappsdk\1.4.230822000\lib\net6.0-windows10.0.18362.0\Microsoft.WinUI\Themes\generic.xaml
````
